.github/workflows/continuous_integration.yml

```diff
@@ -32,6 +32,26 @@ jobs:
         run: fishtape tests/*/*
         shell: fish {0}
 
+  syntax-check:
```

Comment:
I know this is a bit out-of-scope, but is there any significance to the order the jobs are defined and run? I'm curious if we'd want the syntax check to run first before the tests, because obviously if the syntax check fails, the tests will as well. But I forget if GitHub actions will stop running the rest of the tests if one fails, or if the order defined = order run. Do you know?

Reply:
No. In a matrix, one job fail will cause the others to cancel. But otherwise, no, jobs do not effect each other. 

Source: https://github.com/IlanCosman/tide/actions/runs/604243138

Reply:
Good to know, thanks!